README.md

```diff
@@ -17,140 +33,81 @@ Run `make update` to update the bindata. For example, you can see the bindata ha
 ```console
 $ git status
 	modified:   test/extended/testdata/bindata.go
-	new file:   test/extended/testdata/olm/etcd-subscription-manual.yaml
+	new file:   test/extended/testdata/kcp/xxxx.yaml
 ```
 
 ### Compile the executable binary
 Note that we use the `go module` for package management, the previous `go path` is deprecated.
 ```console
-$ git clone git@github.com:openshift/openshift-tests-private.git
-$ cd openshift-tests-private/
+$ git clone git@github.com:kcp-dev/kcp-tests.git
+$ cd kcp-tests/
 $ make build
 mkdir -p "bin"
-export GO111MODULE="on" && export GOFLAGS="" && go build -o "bin" "./cmd/extended-platform-tests"
-$ ls -hl ./bin/extended-platform-tests 
--rwxrwxr-x. 1 cloud-user cloud-user 165M Jun 24 22:17 ./bin/extended-platform-tests
+export GO111MODULE="on" && export GOFLAGS="" && go build  -ldflags="-s -w" -mod=mod -o "bin" "./cmd/kcp-tests"
+$ ls -hl ./bin/kcp-tests 
+-rwxrwxr-x. 1 cloud-user cloud-user 106M Sep 13 14:41 ./bin/kcp-tests
```

Comment:
This section should be after contribution

Reply:
Addressed.